tests/pages/api/definePassengerType.test.ts

```diff
@@ -433,4 +434,24 @@ describe('definePassengerType', () => {
             Location: '/defineTimeRestrictions',
         });
     });
+
+    it('should save the passenger types', async () => {
```

Comment:
should we have a unit test to say that it doesn't save a group passenger type?

Reply:
or, should I say, a passenger type which is part of a group

Reply:
Good shout, added